rest/data/commit_queue.go

```diff
@@ -108,7 +111,7 @@ func (pc *DBCommitQueueConnector) AddPatchForPr(ctx context.Context, projectRef
 	// populate tasks/variants matching the commitqueue alias
 	projectConfig.BuildProjectTVPairs(patchDoc, patchDoc.Alias)
 
-	if err = addMergeTaskAndVariant(patchDoc, projectConfig, &projectRef, commitqueue.SourcePullRequest); err != nil {
+	if err = units.AddMergeTaskAndVariant(patchDoc, projectConfig, &projectRef, commitqueue.SourcePullRequest); err != nil {
```

Comment:
it seems odd to me that this is part of the units package; I see why it can't be part of rest/data but maybe model? Or why do we no longer just want it to be done as part of that job?

Reply:
it can't go in model because it uses the validator :( I didn't see a better existing package for this to go in